webtransport/quic/client-indication.any.js

```diff
@@ -0,0 +1,31 @@
+// META: script=/common/get-host-info.sub.js
```

Comment:
```suggestion
// META: quic=true
// META: script=/common/get-host-info.sub.js
```

(This has no effect now but will be used to differentiate the tests.)